test/powershell/engine/Basic/NativeCommandErrorHandling.Tests.ps1

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Functional tests to verify that errors (non-terminating and terminating) appropriately
```

Comment:
Minor nit - this would read better if it said "Functional tests to verify that native exes error (non-terminating and terminating) appropriately ..."

Reply:
That said, not sure it's worth it since we'll then have to spin the "build wheel" again to see if we get a successful build.  :-)

Reply:
```suggestion
# Functional tests to verify that native executables throw errors (non-terminating and terminating) appropriately
```